docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/ui/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Managing Data Product using Console"
+date: "2024-01-18"
+sidebar_label: "Using the UI"
+sidebar_position: 1
+sidebar_custom_props:
+  offerings:
+    - enterprise
+    - cloud
+
+---
+
+## Creating and editing a new Data Product using Console
+
+To create a new data product, navigate to the "Data products" option in the left main menu and click the "Create data product" button. 
+
+![Create data product](images/image-1.png)
+
+A form will appear on the page. Enter your data product information and click "Create and continue" to navigate to the event specification page.
+
+![Event specifications](images/image-2.png)
+
+This page allows you to create multiple event specifications. You can click on any row to enter the details on this screen, or you can complete the information later.
+
+When clicking on a event specification row, a page will allow you to enter additional information into separate modals:
+
+- **Event information**; describes information such as which applications the event fires in
+- **Event data structure**; determines how your data is structured and define the types of properties of the event
+- **Entity data structures**; describes which entities to attach to the event when it is triggered
+- **Event triggers**; specify the places and circumstances under which the event is triggered
+- **Implementation instructions**; specify instructions about how each schema property should be set for this scenario
```

Comment:
`Implementation instructions` has now been changed to `Properties` so we should update it here. Also we should replace `scenario` with `event specification`.

Reply:
done